pom.xml

```diff
@@ -50,7 +50,7 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <project.systemvm.template.location>https://download.cloudstack.org/systemvm</project.systemvm.template.location>
-        <project.systemvm.template.version>4.18.1.0</project.systemvm.template.version>
+        <project.systemvm.template.version>4.19.0.0</project.systemvm.template.version>
```

Comment:
Fixed @shwstppr wait before kicking packaging, I need to build/scp the templates to d.c.o server.